### ISSUE TITLE
how can we run it?

### ISSUE THREAD

**Issue by fucekar** [dommilosz/Among-us-but-3D#2]
how can we run it?

**Comment by dommilosz** [comment]
Open in unity and click play

**Comment by itayo** [comment]
What version is this written in?
It doesn't seem to be compatible with LTS 2019.4 nor 2018.4

**Comment by dommilosz** [comment]
I'm using 2020.1.11f1

**Comment by kuskus-39** [comment]
helo, how can i run it without unity, i only have visla studio 2019 comunity /s

**Comment by dommilosz** [comment]
I don't think you can run it without unity. https://gamedev.stackexchange.com/questions/183270/build-unity3d-project-from-source-without-unity-editor

**Comment by kuskus-39** [comment]
Thanks! I'll try getting it soon.

**Comment by dommilosz** [comment]
I updated unity to 2021.1.13f1 so make sure you use the same version
